web/src/lib/domain-logic/getContentWithCardCount.ts

```diff
@@ -0,0 +1,30 @@
+import { filterCertifications } from "@/lib/domain-logic/filterCertifications";
```

Comment:
[pebble] I like that we split this out from `TwoTabDashboardLayout`. Given that this isn't specific to that component any more, can we pull out the associated tests as well?